src/components/Tags.tsx

```diff
@@ -20,122 +25,114 @@ const KeyCodes = {
   enter: 13,
 };
 
-type tagstruct = {
```

Comment:
If possible it is always best to use an existing type. You can explore the existing shared types here: https://github.com/cusac/mbt.guide/tree/master/src/types